integrations/great_expectations/fraud_batch_data_validation.ipynb

```diff
@@ -623,49 +656,109 @@
     {
       "cell_type": "markdown",
       "source": [
-        "On the other hand, setting the ingestion policy to \"STRICT\" for the window_agg_fg lead to the abortion of the insertion job. You can however bypass the validation if needed (e.g. time critical backfilling)"
+        "You can also retrieve validation report summaries via the python client."
       ],
       "metadata": {
-        "id": "5yxIaTzNPvLG"
+        "id": "YSFJmZjjwXl8"
       },
-      "id": "5yxIaTzNPvLG"
+      "id": "YSFJmZjjwXl8"
     },
     {
       "cell_type": "code",
       "source": [
-        "trans_fg.insert(trans_df.head(5), validation_options={\"run_validation\": False})"
+        "trans_all_validation_report_summaries = trans_fg.get_all_validation_reports()"
       ],
       "metadata": {
-        "id": "cGq__rjlLw7P"
+        "id": "GsRZZaJkwWzU"
       },
-      "id": "cGq__rjlLw7P",
+      "id": "GsRZZaJkwWzU",
       "execution_count": null,
       "outputs": []
     },
     {
       "cell_type": "markdown",
       "source": [
-        "You can also retrieve validation report summaries via the python client. Feature Groups provide a convenience method to get the latest summary uploaded for quick prototyping."
+        "\n",
+        "You are however more likely to be in one of two use cases: \n",
+        "- Check the latest validation report because something went horribly wrong and you have been tasked with sorting it out.\n",
+        "- Check the history for a particular Expectation to understands exactly when things started to go horribly wrong.\n",
+        "\n",
+        "Luckily Hopsworks provide convenience methods for both cases 🙂"
       ],
       "metadata": {
-        "id": "YSFJmZjjwXl8"
+        "id": "vCG8Wt2NfPCZ"
       },
-      "id": "YSFJmZjjwXl8"
+      "id": "vCG8Wt2NfPCZ"
     },
     {
       "cell_type": "code",
       "source": [
-        "trans_all_validation_report_summaries = trans_fg.get_all_validation_reports()\n",
-        "\n",
         "last_trans_summary = trans_fg.get_latest_validation_report()"
       ],
       "metadata": {
-        "id": "GsRZZaJkwWzU"
+        "id": "4xy1e4dzfXKN"
       },
-      "id": "GsRZZaJkwWzU",
+      "id": "4xy1e4dzfXKN",
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Create a quick history\n",
+        "from time import sleep\n",
+        "from datetime import datetime\n",
+        "\n",
+        "start_time = datetime.now()\n",
+        "\n",
+        "for i in range(0, 10):\n",
+        "  job, report = trans_fg.insert(trans_df[i*10:(i+1)*10], write_options={\"wait_for_job\": False})\n",
+        "  sleep(1) # Force sleep to avoid uploading report with same timestamp\n",
```

Comment:
It's really that fast?

Reply:
Ja, there are not that many requests being made and it is a 10 rows DataFrame so...